packages/codec/lib/compiler/types.ts

```diff
@@ -7,5 +7,5 @@ export interface CompilerVersion {
 
 //NOTE: Families 0.5.0 and up will be named by the lowest version that
 //fits in the given family.  So e.g. 0.5.x covers 0.5.x-0.7.x;
-//0.8.x covers 0.8.x-current (as I write this)
-export type SolidityFamily = "unknown" | "pre-0.5.0" | "0.5.x" | "0.8.x";
+//0.8.x covers 0.8.0-0.8.6; 0.8.7+ covers 0.8.7 to current (as I write this)
+export type SolidityFamily = "unknown" | "pre-0.5.0" | "0.5.x" | "0.8.x" | "0.8.7+";
```

Comment:
`0.8.7+` seems like an odd semver, unless it's our convention?

Reply:
It's not any sort of convention; it's not even really semver.  If you look at this list of "solidity families", you'll see that the whole thing is pretty ad-hoc. :shrug:  I can change it if you think that would be helpful, since obviously this is all prety arbitrary.

Reply:
No, it's fine, there's no need to change it. The code comment now makes sense! It was surprising to see that's all.